.github/workflows/calypso-live.yml

```
@@ -0,0 +1,45 @@
+name: Calypso Live
+
+on:
+  pull_request:
+    types: ['opened']
+
+jobs:
+  calypso-live:
+    name: 'Launch a Calypso.live instance for your branch'
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name
+    timeout-minutes: 10
+
+    steps:
+      - name: Extract branch name
+        run: |
+          echo ::set-output name=BRANCH::${{ github.event.pull_request.head.ref }}
+        id: extract_branch
+
+      - name: Build Calypso.live link.
+        run: |
+          echo '::set-output name=LINK::https://calypso.live/?branch=${{ steps.extract_branch.outputs.BRANCH }}'
+        id: build_link
```

IMO, we should just inline the branch:

```suggestion
    steps:
      - name: Build Calypso.live link.
        run: |
          echo '::set-output name=LINK::https://calypso.live/?branch=${{ github.event.pull_request.head.ref }}'
        id: build_link
```

sidenote: it's unfortunate how sharing variables is so verbose

---

Yeah, you're right, we can keep this short.